lib/oprt.gd

```diff
@@ -1799,7 +1800,19 @@ OrbitsishOperation( "IsTransitive", OrbitsishReq, false, NewProperty );
 ##  <P/>
 ##  <Index>primitive</Index>
 ##  An action is <E>primitive</E> if it is transitive and the action admits
-##  no nontrivial block systems. See&nbsp;<Ref Sect="Block Systems"/>.
+##  no nontrivial block systems. See&nbsp;<Ref Sect="Block Systems"/> for
+##  the definition of block systems.
+##  <P/>
+##  For permutation groups, the syntax <C>IsPrimitive(<A>G</A>)</C> is also
+##  permitted and tests whether the group is primitive on the points moved
+##  by it, that is the group <M>\langle (2,3,4),(2,3) \rangle</M>
+##  is primitive (on 3 points).
+##  <P/>
+##  For an explanation of the meaning of all the inputs, please refer to
+##  &nbsp;<Ref Sect="About Group Actions"/>.
+##  <P/>
+##  <E>Note:</E> This operation does not tell whether a matrix group
+##  preserves a direct sum of vector spaces.
```

Comment:
It never would, but as a naive user you could think that IsPrimitive for a matrix group actually tells you whether this group is a primitive matrix group. But this is not the case.

Reply:
OK, I did not make this connection, so this paragraph, which is meant as a clarification, to me was instead confusing :/. Perhaps it could be changed to something more like this: "This operation is not concerned with the notion of "primitive vector space", i.e., it does not..."